Source/TableView/Implementation/PlainTableView.swift

```diff
@@ -38,11 +38,12 @@ open class PlainTableView<CELL: UIView>: TableViewBase<CELL.StateType, PlainTabl
     public init(
         cellFactory: @escaping () -> CELL = CELL.init,
         style: UITableViewStyle = .plain,
-        reloadable: Bool = true)
+        reloadable: Bool = true,
+        automaticallyDeselect: Bool = true)
     {
         self.cellFactory = cellFactory
 
-        super.init(style: style, reloadable: reloadable)
+        super.init(style: style, reloadable: reloadable, automaticallyDeselect: automaticallyDeselect)
```

Comment:
Is there a chance that there might be more options added in the future? We should ponder whether not to create a dedicated `OptionSet` instead for both `reloadable` and `automaticallyDeselect`.

Reply:
Not sure about adding any more in the future. But please add it to issues. The problem with your proposal is that it might be a problem to implement it without breaking API.

Reply:
@MatyasKriz Great point! Something we should keep an eye out for Reactant 2.0.

Or we could have the `automaticallyDeselect` as a property on the TableView instead of an init parameter. But I'm not sure whether it's required up front or not.